### ISSUE TITLE
[RFC] Handle pre/post plugins per cfg

### ISSUE THREAD

**Issue by narasimhan-v** [open-power-host-os/tests#213]
How about handling pre/post plugin scripts (right not they are only for pip installation) mapped with cfg files ?

We can maintain a list of scripts in a directory, and use them only if needed when a cfg needs it.
We need to additionally maintain a mapping of cfg to scripts, and avocado-setup.py needs to handle it.

Example:
```
[io_nvme]
pre = <pre_script_path/name within wrapper>
post = <post_script_path/name within wrapper>
```

Additionally, in future we may have profiler commands, files also added.

**Comment by PraveenPenguin** [comment]
@narasimhan-v yes I agree with the proposal as the wrapper is tightly coupled with misc test and avocado so at least what our Test require (in this pip package)  we need part of wrapper for maintenance point  it will be good that we can make sure our test should not fail, and avoid to run every time those test cases handling in cfg will really help

**Comment by PraveenPenguin** [comment]
or other  way that we can enable this option when needed can be used as a flag like
https://github.com/open-power-host-os/tests/pull/208 but that is not cfg based it is for the entire run
